### PR TITLE
Issues/69 parent relation for folders

### PR DESCRIPTION
**Pull request by vallo87**
Folder's view now handles single parent select with tree-view component in single-choice mode1<

### CODE REVIEW

File: src/Controller/ModulesController.php [hunk 1]
```diff
@@ -366,6 +366,12 @@ public function relationshipsJson($id, string $relation) : void
             }
 
             $response = $this->apiClient->get($available);
+
+            foreach ($response['data'] as $index => $obj) {
+                if ($obj['id'] == $id) {
+                    array_splice($response['data'], $index, 1);
```
Comment (didoda) [inline src/Controller/ModulesController.php]: Why removing data from `$response['data']`? a comment would help in here ;)

File: webroot/js/app/components/tree-view/tree-view.js [hunk 2]
```diff
@@ -131,6 +142,35 @@ Vue.component('tree-view', {
             this.pendingRelations = this.pendingRelations.filter(pending => pending.id !== related.id);
         },
 
+        /**
+         * remove all related objets from prendingRelations Array
```
Comment (didoda) [inline webroot/js/app/components/tree-view/tree-view.js]: typo: `objects'. `prendingRelations` or `pendingRelations`?

File: webroot/js/app/components/tree-view/tree-view.js [hunk 3]
```diff
@@ -115,11 +126,11 @@ Vue.component('tree-view', {
         },
 
         /**
-         * remove an object from prendingrelations Array
+         * remove an object from prendingRelations Array
```
Comment (didoda) [inline webroot/js/app/components/tree-view/tree-view.js]: `prendingRelations` or `pendingRelations`?

File: src/Controller/ModulesController.php (hunk 1 above)
Comment (vallo87) [reply src/Controller/ModulesController.php]: right, i'll comment it (anyway is cause in folder tree i want to remove the element at index $index, which is the current object)

**Pull request merged**